travis.sh

```diff
@@ -18,12 +18,16 @@ set -x
 # Set pipefail so that `egrep` does not eat the exit code.
 set -o pipefail
 
+GOOGLE_CLOUD_SDK_ROOT="$(gcloud --format='value(installation.sdk_root)' info)"
+
 mvn --batch-mode clean verify | egrep -v "(^\[INFO\] Download|^\[INFO\].*skipping)"
 
 # Run tests using App Engine local devserver.
 test_localhost() {
-  git clone https://github.com/GoogleCloudPlatform/java-repo-tools.git
-  ./java-repo-tools/scripts/test-localhost.sh appengine .
+  if [[ ! -d java-repo-tools ]]; then
+		git clone https://github.com/GoogleCloudPlatform/java-repo-tools.git
+	fi
+  ./java-repo-tools/scripts/test-localhost.sh appengine:run . -- -DcloudSdkPath="$GOOGLE_CLOUD_SDK_ROOT"
```

Comment:
There's a bug in the plugin. Even though the `gcloud` command is on the $PATH in compute engine instances (for the Travis build), it can't find the SDK root.
https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/267

Once that is fixed, the `-DcloudSdkPath` property will not be needed.